src/index.html

```diff
@@ -1,13 +1,373 @@
+          </h2>
+
+          <div class="location gallery__location">
+            <div class="location__images">
+              <div class="location__image--container grid__item--tablet--1-4 grid__item--1-8">
+                <img
+                  src="images/photos/gallery1.jpg"
+                  alt="location image"
+                  class="location__image location__image--1"
+                >
+              </div>
+              <div class=" location__image--container grid__item--tablet--5-6 grid__item--9-12">
+                <img
+                  src="images/photos/gallery2.jpg"
+                  alt="location image"
+                  class="location__image location__image--2"
+                >
+              </div>
+            </div>
+            <div class="location__description">
+              <h3>Headphones & earbuds</h3>
+            </div>
+          </div>
+
+          <div class="location gallery__location">
+            <div class="location__images">
+              <div class="location__image--container grid__item--tablet--1-3 grid__item--1-6">
+                <img
+                  src="images/photos/gallery3.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+              <div class="location__image--container grid__item--tablet--4-6 grid__item--7-12">
+                <img
+                  src="images/photos/gallery4.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+            </div>
+            <div class="location__description">
+              <h3>Speakers</h3>
+            </div>
+          </div>
+
+          <div class="location gallery__location">
+            <div class="location__images">
+              <div class="location__image--container grid__item--tablet--1-2 grid__item--1-4">
+                <img
+                  src="images/photos/gallery5.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+              <div class="location__image--container grid__item--tablet--3-6 grid__item--5-12">
+                <img
+                  src="images/photos/gallery6.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+            </div>
+            <div class="location__description">
+              <h3>Audio sunglasses</h3>
+            </div>
+          </div>
+
+          <a
+            href="#"
+            class="button button--view"
+          >
+            View all
+          </a>
+        </div>
+      </section>
+
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="about__content">
+            <h2 class="about__title grid__item--tablet--1-3">
+              Why buy direct from Bose
+            </h2>
```

Comment:
The gap is too wide , please check the figma design
and `why buy direct` this txt and then line break as per design
<img width="781" alt="image" src="https://user-images.githubusercontent.com/100297341/194241955-9684283e-f8da-4cd3-81bb-2348051a666a.png">